ndscan/plots/image_2d.py

```diff
@@ -150,14 +150,14 @@ def _update(self):
 
         x_inds = _coords_to_indices(x_data[num_skip:num_to_show], self.x_range)
         y_inds = _coords_to_indices(y_data[num_skip:num_to_show], self.y_range)
-
-        z_min, z_max = self.current_z_limits
-        z_scaled = (z_data[num_skip:num_to_show] - z_min) / (z_max - z_min)
+        for x, y, z in zip(x_inds, y_inds, z_data):
```

Comment:
Seems like this should also take `z_data[num_skip:num_to_show]`?

Reply:
Ah yes, whoops!